Server/src/migrations/SeedDatabase.ts

```diff
@@ -308,6 +308,7 @@ export class SeedDatabase1611344612000 implements MigrationInterface {
     await connection.manager.save(datapoint9);
 
     await queryRunner.query('INSERT INTO accounts_datasets_dataset (accountsId, datasetId) VALUES (1, 2)');
+    await queryRunner.query('ALTER TABLE savedgraphs ADD COLUMN axisZoomEnd JSON NULL AFTER axisZoomStart, CHANGE COLUMN axisMode axisLog JSON NULL DEFAULT NULL, CHANGE COLUMN axisZoom axisZoomStart JSON NULL DEFAULT NULL');

```

Comment:
are you sure we should alter and not have plain insert? I assume above in the code is insert savedGraphs but wrong schema?

Reply:
This migration line is temporary. When it's ready for merge I will take out these lines and alter the DB directly.

Reply:
ok gotcha